server/types/varchar.go

```diff
@@ -77,6 +78,26 @@ func (b VarCharType) CollationCoercibility(ctx *sql.Context) (collation sql.Coll
 	return sql.Collation_binary, 5
 }
 
+func (b VarCharType) CharacterSet() sql.CharacterSetID {
+	return sql.CharacterSet_binary // TODO
+}
+
+func (b VarCharType) Collation() sql.CollationID {
+	return sql.Collation_Default // TODO
+}
+
+func (b VarCharType) MaxCharacterLength() int64 {
+	return b.Length() // TODO
+}
+
+func (b VarCharType) MaxByteLength() int64 {
+	return b.Length() * 4 // TODO
+}
+
+func (b VarCharType) Length() int64 {
+	return int64(b.MaxChars)
+}
```

Comment:
To follow along with every other type, we should add function comments for these, and order them alphabetically